.changeset/fruity-sites-boil.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/runtime-definitions": minor
+---
+---
+"section": other
+---
+
+ContainerRuntime's 'batchBegin'/'batchEnd' events: Removing the `contents` property on event arg `op`
+
+The 'batchBegin'/'batchEnd' events on ContainerRuntime indicate when a batch is beginning/finishing being processed.
+The `contents` property on there is not useful or relevant when reasoning over incoming changes at the batch level.
+So it has been removed from the `op` event arg.
```

Comment:
```suggestion
Changes to the batchBegin and batchEnd events on ContainerRuntime

The 'batchBegin'/'batchEnd' events on ContainerRuntime indicate when a batch is beginning or finishing being processed. The `contents` property on the event argument `op` is not useful or relevant when reasoning over incoming changes at the batch level. Accordingly, it has been removed from the `op` event argument.
```

Reply:
@tylerbutler you'll have to make a follow-up PR, Kian had auto-merge on. I'm still OOF just saw the notification

Reply:
No worries, I'll chat with Kian about it.

Reply:
https://github.com/microsoft/FluidFramework/pull/23015